File: FrontEnd/ideaevaluationfe/src/app/models/judge.ts
import { Idea } from "./idea";

export class Judge{
     name:string;
     emailid:string;
     judgeid:number;
     assignedIdeas: Idea[];
    
    constructor(name : string, emailid : string, judgeid : number, ideas : Idea[]){
        this.emailid = emailid;
        this.name = name;
        this.judgeid = judgeid;
        this.assignedIdeas = ideas;
    }
}

File: IdeaEvaluation/IdeaEvaluation/Models/Judge.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace IdeaEvaluation.Models
{
    public class Judge
    {
        public int JudgeId { get; set; }
        public string Name { get; set; }
        public string EmailId { get; set; }
        public List<Idea> AssignedIdeas { get; set; }
    }
}